.env

```diff
@@ -59,6 +59,7 @@ CPB_ACCOUNT_MANAGER=""
 ###> GraphicServiceOrderBundle ###
 # Form configuration
 FORM_FILE_GS_UPLOAD_SIZE=
+GS_FILES_DIRECTORY=
```

Comment:
Consider adding a reasonable default, e.g. `GS_FILES_DIRECTORY="%kernel.project_dir%/files"`. If you add this, remember to resolve the path in `services.yaml` (cf. https://symfony.com/doc/current/configuration/env_var_processors.html):

```yaml
parameters:
    gs_files_directory: '%env(resolve: GS_FILES_DIRECTORY)%'
```

Reply:
Fixed